core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -58,7 +58,7 @@
 @EqualsAndHashCode(callSuper = false)
 public class GenericContainer<SELF extends GenericContainer<SELF>>
         extends FailureDetectingExternalResource
-        implements Container<SELF> {
+        implements Container<SELF>, AutoCloseable {
```

Comment:
small non-related change to simplify the testing

Reply:
FTR IntelliJ IDEA is barking about this, see https://imgur.com/a/sD1WNyG ;)